tests/devpool-issue-handler.test.ts

```diff
@@ -1235,11 +464,74 @@ describe("handleDevPoolIssue", () => {
       }) as GitHubIssue;
 
       expect(updatedIssue).not.toBeNull();
-      expect(updatedIssue?.state).toEqual("closed");
-
-      expect(logSpy).not.toHaveBeenCalledWith(expect.stringContaining(`Updated state`));
+      expect(updatedIssue.state).toEqual("open");
+      expect(updatedIssue.labels).not.toEqual(
+        expect.arrayContaining([
+          expect.objectContaining({
+            name: "Unavailable",
+          }),
+        ])
+      );
     });
   });
+
+  test("getRepoUrls", async () => {
+    let orgOrRepo = "test/org/bad-repo/still/bad";
+    const warnSpy = jest.spyOn(console, "warn").mockImplementation();
+    await getRepoUrls(orgOrRepo);
+
+    expect(warnSpy).toHaveBeenCalledWith(`Neither org or nor repo GitHub provided: test/org/bad-repo/still/bad.`);
+
+    orgOrRepo = "/test";
+
+    await getRepoUrls(orgOrRepo);
+
+    expect(warnSpy).toHaveBeenCalledWith(`Invalid org or repo provided: `, orgOrRepo);
+
+    (orgOrRepo as any) = undefined;
+
+    await getRepoUrls(orgOrRepo);
+
+    expect(warnSpy).toHaveBeenCalledWith(`No org or repo provided: `, orgOrRepo);
+
+    warnSpy.mockClear();
+
+    jest.resetModules();
+
+    const errorSpy = jest.spyOn(console, "error").mockImplementation();
+    (orgOrRepo as any) = ".";
+
+    await getRepoUrls(orgOrRepo);
+
+    const localErr = `Getting ${orgOrRepo} org repositories failed: HttpError: Bad credentials`;
+    const githubErr = `Getting ${orgOrRepo} org repositories failed: HttpError: Not Found`;
+
+    if (warnSpy.mock.calls.length > 0) {
+      const errThrown: string = warnSpy.mock.calls.flatMap((call) => call).includes(localErr) ? localErr : githubErr;
+      if (errThrown.includes("Bad credentials")) {
+        expect(errThrown).toEqual(localErr);
+      } else {
+        expect(errThrown).toEqual(githubErr);
+      }
+    }
+    expect(errorSpy).toHaveBeenCalledWith(expect.objectContaining({ status: 404 }));
+
```

Comment:
This test is failing for me locally.

```
handleDevPoolIssue › getRepoUrls

    expect(jest.fn()).toHaveBeenCalledWith(...expected)

    Expected: ObjectContaining {"status": 404}
    Received: [HttpError: Bad credentials]

    Number of calls: 1

      515 |       }
      516 |     }
    > 517 |     expect(errorSpy).toHaveBeenCalledWith(expect.objectContaining({ status: 404 }));
          |                      ^
      518 |
      519 |     (orgOrRepo as any) = "-/test";
      520 |     await getRepoUrls(orgOrRepo);

      at Object.<anonymous> (tests/devpool-issue-handler.test.ts:517:22)
```

Reply:
Did it pass if you run without my changes?

Reply:
Yes, before recent changes all tests were passing.

Reply:
Not sure what I can do here as I can't reproduce this issue. It works on my machine and on PR build..

Reply:
If it works on ci then it's fine.

Reply:
Okay then. I will mark it this resolved